### DISCUSSION TITLE
Theme change when you cross your pb speed

### DISCUSSION THREAD

**Discussion by saayan007s** [monkeytypegame/monkeytype#4715] [Ideas]
As you cross your pb speed, the theme will change. It could change in different ways (depending on the user and of course, whether it is possible to implement). Here are some ideas

- simple smooth transition from one theme to another (perhaps from a light to a dark theme or vice versa)
- continuously switching between 2 themes (kind of like the flashlight on top of police cars), this could be accompanied by sounds
- it could be based only sounds and no themes, one type of sound to indicate you became faster than your pb and one to tell you you've slowed down
- another idea is playing around with funboxes (eg "earthquake" when you cross your pb); not sure if this is possible since turning off the "earthquake" funbox mid test restarted the test for me

What do you people think of this?

**Comment by Mushroomportabala007** [comment]
Interesting.
